src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,24 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.NoSuchElementException;
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDao actorDao;
```

Comment:
```suggestion
    private final ActorDao actorDao;
```
not that important for this task, but still
same for other services